pkg/operator/sync.go

```diff
@@ -927,6 +928,66 @@ func (optr *Operator) safetySyncControllerConfig(config *renderConfig) error {
 // the operator version and controller version match. We can call it from safetySyncControllerConfig
 // because safetySyncControllerConfig ensures that the operator and controller versions match before it syncs.
 func (optr *Operator) syncControllerConfig(config *renderConfig) error {
+	editCCAnno := false
+	kubeConfigData, err := optr.clusterCmLister.ConfigMaps("openshift-config-managed").Get("kube-apiserver-server-ca")
```

Comment:
note (non-blocking): We should consider making `"openshift-config-managed"` a constant somewhere.

(That is outside the scope of this PR though)